### ISSUE TITLE
add alt text to all figures on the website

### ISSUE THREAD

**Issue by isabelizimm** [posit-dev/positron#9796]
Currently, any alt text shows up in the rendered pages as regular text. We'll want to add descriptions throughout the site, and fix up the styling so they either are portrayed differently on the page or show up in site metadata only.

**Comment by juliasilge** [comment]
What we actually need here is the metadata description.

**Comment by juliasilge** [comment]
To confirm we have these set now, you should be able to check out our pages with a social share tester such as https://socialsharepreview.com

We do not yet have a default social sharing image, but I have a request in to marketing to give us one to use.

**Comment by juliasilge** [comment]
I added the social share image as well, so now you should get that in previews too.

- If a page is like https://positron.posit.co/quarto.html and has an image on it, that image from the page is used in the preview.
- If a page is like https://positron.posit.co/interpreter-startup.html and do not have an image on it, the default social share image is used in the preview.

**Comment by testlabauto** [comment]
### Verified Fixed
    Positron Version(s) : 2026.01.0-98
    OS Version          : OSX

### Test scenario(s) 

Check various pages with https://socialsharepreview.com/
Looks good.

### Link(s) to test cases run or created:
